src/status_im/ui/screens/desktop/main/chat/views.cljs

```diff
@@ -124,7 +124,7 @@
   (if (= type :datemark)
     ^{:key (str "datemark" message-id)}
     [message.datemark/chat-datemark value]
-    (when (= content-type constants/text-content-type)
+    (when (some #{content-type} [constants/text-content-type constants/content-type-emoji])
```

Comment:
I think you can just invert this:
`(#{constants/text-content constants/content-type-emoji} content-type)`
in order to avoid having to iterate over

Reply:
That would work, too. Again, would make sense to have this set as a constant